test/steps/kogitoapp.go

```diff
@@ -102,7 +102,6 @@ func registerKogitoAppSteps(ctx *godog.ScenarioContext, data *Data) {
 
 	// DeploymentConfig steps
 	ctx.Step(`^Kogito application "([^"]*)" has (\d+) pods running within (\d+) minutes$`, data.kogitoApplicationHasPodsRunningWithinMinutes)
```

Comment:
Should we change this to `Kogito Runtime` ?

Reply:
Will remove the KogitoApp parts as suggested above.